docs/API.md

```diff
@@ -89,7 +89,7 @@ $ curl -X POST --header "Content-Type:application/json" \
 
 > **Note:** on devices with supervisor version lower than 7.22.0, replace all `BALENA_` variables with `RESIN_`, e.g. `RESIN_SUPERVISOR_ADDRESS` instead of `BALENA_SUPERVISOR_ADDRESS`.
 
-Triggers an update check on the supervisor. Optionally, forces an update when updates are locked.
+Triggers an update check on the supervisor. Optionally, forces an update when updates are locked. This update check will check for service updates and will not update the supervisor itself.
```

Comment:
I think the whole phrase is a bit long and not too clear. How about something like  
```suggestion
Triggers a check for the target state of configurations and app services. Optionally, if a new target is available, the `force` options forces the supervisor to ignore locks and install the update. 
```

@cywang117 what do you think?

Reply:
@pipex I'd like to have somewhere the explicit information that his endpoint will not trigger the supervisor to update itself.

Reply:
Fair enough. Let's add something like `This will NOT update the supervisor service.`.